endpoints/citrus-camel/src/main/java/com/consol/citrus/camel/actions/StartCamelRouteAction.java

```diff
@@ -29,7 +29,7 @@
 public class StartCamelRouteAction extends AbstractCamelRouteAction {
 
     /** Logger */
-    private static Logger log = LoggerFactory.getLogger(StartCamelRouteAction.class);
+    private static final Logger LOG = LoggerFactory.getLogger(StartCamelRouteAction.class);
```

Comment:
just a quick question at this point, didn't we agree on `logger` a while ago? I don't remember it exactly, sorry ;)

Reply:
yes, right we have agreed on something (I do not remember the exact wording). Do we have an issue already to tackle all of them at once?